bash5/streaming_pipe_in.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Script reads text from a named pipe and writes it to stdout, capitalized
+pipe_in="$1"
+
+cat "$pipe_in" | tr '[:lower:]' '[:upper:]'
```

Comment:
```suggestion
tr '[:lower:]' '[:upper:]' < "$pipe_in"
```

This way, it doesn't create another process (`cat`)

Reply:
(also applies to the bash3 version of this script)